rbx_types/src/content.rs

```diff
@@ -55,6 +55,24 @@ impl Content {
     pub fn into_value(self) -> ContentType {
         self.0
     }
+
+    /// Returns Some(&str) if `Content` is a Uri.
+    #[inline]
+    pub fn as_uri(&self) -> Option<&str> {
+        match self {
+            Self(ContentType::Uri(uri)) => Some(uri.as_str()),
+            _ => None,
+        }
+    }
+
+    /// Returns Some(Ref) if `Content` is an Object.
```

Comment:
Same as the other function. This is not a very helpful doc comment since it's essentially just repeating the type signature without any more information.